inference-engine/include/ie_core.hpp

```diff
@@ -185,7 +202,8 @@ class INFERENCE_ENGINE_API_CLASS(Core) {
      * @brief Sets configuration for device, acceptable keys can be found in ie_plugin_config.hpp
      *
      * @param deviceName An optional name of a device. If device name is not specified, the config is set for all the
-     * registered devices.
+     * registered devices. The special case is CACHE_DIR global key. When "CACHE_DIR" is enabled for all devices,
+     * model caching is enabled for all supported devices. Devices which don't support model caching will ignore this config setting
```

Comment:
may be we need to move this "special case" / notes to the `CACHE_DIR` documentation?